modules/installation-osp-setting-worker-affinity.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-installer-restricted.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+
+[id="installation-osp-setting-worker-affinity_{context}"]
+= Setting compute machine affinity
+
+Optionally, you can set the affinity policy for compute machines during installation.
+
+[TIP]
+====
+The installer does not select an affinity policy for compute machines by default. 
```

Comment:
Sugg: s/installer/installation program/

A) That's the language in the "Obtaining the installation program".
B) This is fear-based, but it seems like "installer" could be construed to be the human operator who performs the installation.

Up to you.

Reply:
I've been chastised for using "installation program" in the past. We use "installer" freely in other parts of the docs, e.g. "installer-provisioned infrastructure," so I'm apt to leave it. 

Personally, yeah, I prefer "installation program." :)